### ISSUE TITLE
[explicit-function-return-type] auto fix for adding `public`

### ISSUE THREAD

**Issue by wSedlacek** [typescript-eslint/typescript-eslint#3117]
- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.
- [x] I have [read the FAQ](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/FAQ.md) and my problem is not listed.

**Repro**

```JSON
{
  "rules": {
    "@typescript-eslint/explicit-member-accessibility": ["warn"]
  }
}
```

```TS
export class Example {
  something() {}
}
```

**Expected Result**

I expect an auto fix to be available to add `public` before `something()`

**Actual Result**

No auto fix is available, instead I have to manually add `public`

**Additional Info**

[member-access](https://github.com/palantir/tslint/blob/10fc233b2555044b145c9bbd9f602d43e8df2216/src/rules/memberAccessRule.ts#L202) had a fix for this, it would be nice for `explicit-function-return-type` to fix this as well.

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `4.15.2` |
| `@typescript-eslint/parser`        | `4.15.2` |
| `TypeScript`                       | `4.2.2` |
| `ESLint`                           | `7.21.0` |
| `node`                             | `14.15.4` |

**Comment by bradzacher** [comment]
https://github.com/typescript-eslint/typescript-eslint/issues/503#issuecomment-670612938

Please use the search

**Comment by wSedlacek** [comment]
Thank you for the link! I had done a search through the titles, but I guess I didn't look hard enough.
